queries.sql

```diff
@@ -32,4 +32,51 @@ WHERE name != 'Gabumon';
 
 SELECT  *
 FROM animals
-WHERE weight_kg BETWEEN 10.4 AND 17.3;
+WHERE weight_kg BETWEEN 10.4 AND 17.3; begin; update animals
+
+SET species = 'nonspecified';
+
+SELECT  *
+FROM animals; rollback; begin; update animals
+
+SET species = 'digimon'
+WHERE trim(name) like '%mon'; update animals
+
+SET species = 'pokemon'
+WHERE species is NULL; commit; begin; delete
+FROM animals;
+
+SELECT  *
+FROM animals; rollback;
+
+SELECT  *
+FROM animals; begin; delete
+FROM animals
+WHERE date_of_birth > '01/01/2022'; savepoint neg; update animals
+
+SET weight_kg = (weight_kg * -1); rollback to neg; update animals
+SET weight_kg = (weight_kg * -1)
+WHERE weight_kg < 0; release savepoint neg; commit;
```

Comment:
- [ ] Kindly consider structuring your queries where `BEGIN; UPDATE; COMMIT;` each is placed on a separate line so that the queries look organized and clean.
- [ ] You have not implemented the following part of the requirement and in result the `species` column is null in your table for all rows, so consider fixing it.
- Inside a transaction:
     - Update the animals table by setting the species column to digimon for all animals that have a name ending in 
     mon.
     - Update the animals table by setting the species column to pokemon for all animals that don't have species 
     already set.
     - Commit the transaction.
     - Verify that change was made and persists after commit.
- [ ] Kindly consider implementing the requirement one by one and also check where it asks you to check if some specific change in the table persist or not. and also take screenshot where it says to take a screen shot so that the reviewer can see the result of specific query.